Dockerfile

```diff
@@ -1,10 +1,28 @@
-FROM node:14-slim
+FROM node:14-slim as builder
+
+WORKDIR /usr/src/app
+
+RUN apt-get update && apt-get install -y \
+    python3 \
+    gcc \
+    build-essential \
+    && apt-get clean
+
 COPY package.json package.json
 COPY package-lock.json package-lock.json
-RUN npm install
+
+RUN npm install --production
```

Comment:
`npm ci` might be a better choice here, it's faster and always respects `package-lock.json` 
https://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable